com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/ShaderGraphUIBlock.cs

```diff
@@ -9,34 +9,57 @@
 
 namespace UnityEditor.Rendering.HighDefinition
 {
-    class ShaderGraphUIBlock : MaterialUIBlock
+    /// <summary>
+    /// ShaderGraph material UI block.
+    /// This class will display every non-hidden properties inside a shader and thus it can also be used on non-shadergraph shaders.
```

Comment:
This UI block displays every non-hidden property inside a shader. You can also use this with non-shadergraph shaders.